bigtable-hbase-1.x-parent/bigtable-hbase-1.x-mapreduce/src/test/java/com/google/cloud/bigtable/mapreduce/validation/TestBigtableSyncTableJob.java

```diff
@@ -24,6 +24,24 @@
 // TODO - parameterize this to run against prod in future
 public class TestBigtableSyncTableJob {
 
+  @Test
+  public void testMissingArgs() {
+    String[] args = {
+      "--sourcezkcluster=hbase-m:2181:/hbase",
+      "--targetbigtableproject=prod-app-bigtable-svcs",
+      "--targetbigtableinstance=prod-app-events",
+      "--targetbigtableprofile=default",
```

Comment:
what is this? Is this application profile, if so we should name it that way.

Reply:
is this consistent with other jobs?

Reply:
good suggestions. it's not the prettiest, but updated app profile options to be more precise - sourcebigtableappprofile and targetbigtableappprofile.

Reply:
the naming is consistent with upstream [SyncTable.java#L808-L842](https://github.com/apache/hbase/blob/master/hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/SyncTable.java#L808-L842)